### PR TITLE
Create host.py example for qrng

### LINKED ISSUE #492: Add Python host to QRNG sample
Add a Python host program (host.py) to the QRNG sample in getting-started: https://github.com/microsoft/Quantum/tree/main/samples/getting-started/qrng

### PR DESCRIPTION
**Pull request by perryrivera**
Fixes #492. This PR adds a host.py Python program to run the random number generator from Python.

### CODE REVIEW

**Review by guenp — APPROVED** [review]

**Comment by guenp** [comment]
@perryrivera thanks for your contribution! I've added some more docs to the README. Also, we require contributors to sign our [CLA](https://cla.opensource.microsoft.com/microsoft/Quantum?pullRequest=493) before we can merge.

File: samples/getting-started/qrng/README.md [hunk 1]
```diff
@@ -11,19 +11,21 @@ description: "This sample implements a quantum random number generator using Q#,
...
 
 ## Prerequisites ##
 
 - The Microsoft [Quantum Development Kit](https://docs.microsoft.com/azure/quantum/install-overview-qdk/).
+- The [`qsharp` Python package](https://docs.microsoft.com/en-us/azure/quantum/install-python-qdk?tabs=tabid-conda) (optional for usage with Python host program).
 
 ## Running the Sample ##
 
-To run the sample, use the `dotnet run` command from your terminal. 
+To run the sample, use the `dotnet run` command from your terminal. To run the sample via the python host program, run `python host.py` from your terminal.
```
Comment (cgranade) [inline samples/getting-started/qrng/README.md:25]:
```suggestion
To run the sample, use the `dotnet run` command from your terminal. To run the sample via the Python host program, run `python host.py` from your terminal.
```

File: samples/getting-started/qrng/host.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import qsharp
```
Comment (cgranade) [inline samples/getting-started/qrng/host.py:1]:
```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.

import qsharp
```

File: samples/getting-started/qrng/host.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+from Qrng import SampleQuantumRandomNumberGenerator
+
+print(SampleQuantumRandomNumberGenerator.simulate())
```
Comment (cgranade) [inline samples/getting-started/qrng/host.py:4]:
```suggestion
print(SampleQuantumRandomNumberGenerator.simulate())

```

**Pull request merged**